### ISSUE TITLE
Improve Badge visibility

### ISSUE THREAD

**Issue by jnowaczek** [UMDLARS/dtanm#135]
The Badge system is cool. There should be more places to show off Badges in the UI.

**Comment by pahp** [comment]
We are resurrecting DTANM after a long slumber... I don't even see where there are any badges. Where are they visible?

**Comment by ChandlerSwift** [comment]
Yeah, it shows under teams. Add 'em in the "Edit Teams" dialog box:

![Image](https://github.com/user-attachments/assets/8cf5edb8-dad3-499f-8efe-db021bf56e02)
